pkg/bosh/manifest/container_factory.go

```diff
@@ -156,22 +86,92 @@ func (c *ContainerFactory) JobsToContainers(jobs []Job) ([]corev1.Container, err
 	return containers, nil
 }
 
+func (c *ContainerFactory) generateJobContainers(job Job, jobImage string) ([]corev1.Container, error) {
+	boshJobName := job.Name
+	containers := []corev1.Container{}
+	template := corev1.Container{
```

Comment:
Uhm, while you're at it, could you extract that into a func? I think all the other containers have one (`templateRenderingContainer`, etc...)